docker-compose/postgres.yml

```diff
@@ -0,0 +1,12 @@
+version: '3'
+services:
+  local-database:
```

Comment:
How about moving this service to main `docker-compose.yml` file? It's only supposed to be used for development purposes anyways, so I would say it would be good if `docker-compose.yml` included whole working environment.

Reply:
Good point